### PR TITLE
[Resources] Update Bicep install URL

### PR DESCRIPTION
**Pull request by shenglol**
## Description

The Bicep installation doc has been moved to MS docs. The PR updates the link to the doc.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Resources/ResourceManager/Properties/Resources.resx [hunk 1]
```diff
@@ -501,7 +501,7 @@ You can help us improve the accuracy of the result by opening an issue here: htt
     <value>Unrecognized resource change {0}: {1}. Specify one ore more values in the following list and try again: {2}.</value>
   </data>
   <data name="BicepNotFound" xml:space="preserve">
-    <value>Cannot find Bicep. Please add Bicep to your PATH or visit https://github.com/Azure/bicep/blob/main/docs/installing.md to install Bicep.</value>
+    <value>Cannot find Bicep. Please add Bicep to your PATH or visit https://aka.ms/bicep-install to install Bicep.</value>
```
Comment (BethanyZhou) [inline src/Resources/ResourceManager/Properties/Resources.resx:504]: This change wouldn't take effect in module immediately. Could you add this message to MDs of bicep related cmdlets.

Comment (shenglol) [reply src/Resources/ResourceManager/Properties/Resources.resx:504]: Yeah, it's already backfilled: https://github.com/Azure/bicep/blob/main/docs/installing.md.

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Pull request merged**